app/src/main/res/values/strings.xml

```diff
@@ -471,8 +471,8 @@
         <item>Red Volcano</item>
         <item>Purple Pro</item>
     </string-array>
-    <string name="icon_pack_choose_title">Icon pack</string>
+    <string name="icon_pack_choose_title">Icons</string>
```

Comment:
There will be icons to download manually which will not be integrated as a pack. It helps to differentiate them in the title.

Reply:
I don't understand. "Icons" means there is more than one.

Reply:
It is a set of pre-built icons, there will also be custom icons and icons in the form of plugins, these are not the same elements. Icon pack, it is only for basic icon pack not the others.